scipy/signal/ltisys.py

```diff
@@ -445,6 +423,13 @@ def impulse(self, X0=None, T=None, N=None):
         """
         return impulse(self, X0=X0, T=T, N=N)
 
+    def impulse2(self, X0=None, T=None, N=None, **kwargs):
```

Comment:
We just merged gh-4675, which fixes up ``lsim/step/impulse``. So long-term we'll indeed want to deprecate the ``*2`` versions of those.

I agree that it would be better to not add something here that will be deprecated soon.